src/googmodule.ts

```diff
@@ -573,8 +573,38 @@ export function commonJsToGoogmoduleTransformer(
             ts.createAssignment(ts.createIdentifier('module'), ts.createIdentifier('module'))));
       }
 
-      // Insert goog.module() etc after any leading comments in the source file. The comments have
-      // been converted to NotEmittedStatements by transformer_util, which this depends on.
+      // Add goog.require('tslib');
+      // if this is not JS transpilation, and it hasn't already been required.
+      // Rationale:
+      // TS gets compiled to Development mode (ES5) and Closure mode (~ES6)
+      // sources. Tooling generates module manifests from the Closure version.
+      // These manifests are used both with the Closure version and the
+      // Development mode version. 'tslib' is sometimes required by the
+      // development version but not the Closure version. Inserting the import
+      // below unconditionally makes sure that the module manifests are
+      // identical between Closure and Development mode, avoiding breakages
+      // caused by missing module dependencies.
+      if (!host.isJsTranspilation) {
+        // Need to get a copy of the already resolved module names before
+        // calling resolveModuleName on 'tslib'
```

Comment:
don't document what you're doing, but why you're doing it. E.g. the bit that's missing here is why you need to do this.

Reply:
Done. How does the new version look?